applications/TenetFarm.md

```diff
@@ -0,0 +1,162 @@
+
+
+
+
+
+### Ecosystem Fit 
+Currently, there are no known competitors of Tenet. Tenet is the only whitelabel geyser solution that all DeFi projects can use to quickly spin up incentivized liquidity pools on various protocols across Binance Smart Chain, Ethereum, and Polkadot in the future. Incentives can include both their own native token as well as Tenet's native $TEN token. Other blockchains will be included soon so that the Tenet protocol and its $TEN token will become the first project of its kind - a Layer 2 DeFi connector for all Automatic Market Maker (AMM) platforms.
+Instead of being a competitor, Tenet introduces the liquidity tap + liquidity pool as a market standard in the future to fully guarantee the effectiveness and efficiency of AMMs.
+
+For example, if a project wants to start a Balancer liquidity pool, they can start the pool, incent it with their own token and redirect TEN incentives to it all within our easy-to-use interface at https://eth.tenet.farm/pool/create. We call this idea a Layer 2 AMM connector or Liquidity Tap.
+
+
+## Team :busts_in_silhouette:
+
+### Team members
+* Steve Thung
+* Wade
+* Liang Peng
+
+### Contact
+* **Contact Name:** Steve Thung
+* **Contact Email:** steve@tenet.farm
+*  **Website:** https://www.tenet.farm
+
+### Legal Structure 
+* **Registered Address:** Maystar Chambers, P.O. Box 3269, Apia, Samoa
+* **Registered Legal Entity:** Tenet Finance Limited.
+
+### Team's experience
+Steve Thung, the CTO of Tenet. His charge is to develop smart contracts, design product functionalities, and manage project schedule. So far, Steve has 15 years of trading system development experience and many years of blockchain development experience.
+
+### Team Code Repos
+* https://github.com/TenetFarm/tenet-core
+
+
+## Development Roadmap :nut_and_bolt: 
+
+### Overview
+* **Total Estimated Duration:** 1.5 Month
+* **Full-time equivalent (FTE):**  2 FTE
+* **Total Costs:** 0.8 BTC
+
+### Milestone 1 Polkadot Adaption Development
+* **Estimated Duration:** 1.5 month
+* **FTE:**  2
+* **Costs:** 0.8 BTC
+
+| Number | Deliverable | Specification |
+| ------------- | ------------- | ------------- |
+| 0a. | License | MIT |
+| 0b. | Documentation | Tenet documentation center (Help center) is live, we will provide more inline documentation of the code and a basic tutorial to show how Tenet (Polkadot version) works. |
+| 0c. | Testing Guide | The code will have unit-test coverage (min. 70%) to ensure functionality and robustness. In the guide we will describe how to run these tests | 
+| 0d. | Article/Tutorial | We will write an article or tutorial that explains the work done as part of the grant. |  
+| 1. | Chain Adapter | Has already adapted Polkadot, ETH and BSC|  
```

Comment:
I was mostly referring to the technical details of these deliverables. What’s exactly the adapter and how is it implemented (programming language etc.)?

Reply:
Updated